engine/hatchery/swarm/swarm.go

```diff
@@ -213,12 +221,12 @@ func (h *HatcherySwarm) SpawnWorker(spawnArgs hatchery.SpawnArguments) (string,
 				if network == "" {
 					network = name + "-net"
 					networkAlias = "worker"
-					if err := h.createNetwork(dockerClient, network); err != nil {
+					if err := h.createNetwork(ctx, dockerClient, network); err != nil {
```

Comment:
invalid operation: h (variable of type *HatcherySwarm) has no field or method createNetwork